### DISCUSSION TITLE
Filter GitHub notifications by repository name wildcard filter?

### DISCUSSION THREAD

**Discussion by sheeeng** [community/community#118775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Could we support filtering GitHub notification by repository name wildcard filter?

For example, I would like to filter all notifications from any repository name that contains `azure` under `microsoft` organization.

```text
org:microsoft repo:microsoft/*azure*
```

The above repository name wildcard filter is more readable and usable over the following long listing of exact names of repositories.

```
org:microsoft repo:microsoft/azure-pipelines-tasks  repo:microsoft/azure-pipelines-yaml 
```

**Comment by lelamaltez** [comment]
As of my last update in January 2022, GitHub's notification filtering options are primarily based on the type of notification (e.g., issue, pull request, mention) rather than by repository name wildcard filters. However, you can create email filters or rules in your email client to organize GitHub notifications based on repository names.

Here's how you can set up email filters in some popular email clients:

Gmail:

Open Gmail and locate a notification email from GitHub.
Click the three dots ("More") next to the reply button.
Select "Filter messages like these."
In the filter dialog, specify criteria such as "From," "To," or "Subject" to match GitHub notifications.
Click "Create filter."
Check "Apply the label" and select or create a label for GitHub notifications.
Click "Create filter."

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @sheeeng !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
